### ISSUE TITLE
"Select" button no longer appears in image field

### ISSUE THREAD

**Issue by marco-land** [sanity-io/sanity#6999]
The "Select" button no longer appears in the image field box. This problem occurs in `3.47.0` and above.

**To Reproduce**

Add an image field
```js
{
      title: 'Thumbnail',
      type: 'image',
      name: 'thumbnail',
},
```

**Screenshots**
![CleanShot 2024-06-20 at 10 27 57@2x](https://github.com/sanity-io/sanity/assets/24410335/0d234ada-7114-4c90-87e5-5609b1339cca)

**Which versions of Sanity are you using?**

`3.47.1` I also tested `3.47.0`

**Comment by notflip** [comment]
@marco-land I don't have the media-plugin and I'm not seeing the "Select" button, **how did you get it back?**

**Comment by r0skar** [comment]
I am seeing the same behavior as @notflip. Not using the media plugin and a basic field config:

```
    defineField({
      name: "cover",
      type: "image",
      options: {
        hotspot: true,
      },
    }),
```

**Comment by notflip** [comment]
@marco-land can you reopen the ticket please?

**Comment by HumidBrains** [comment]
I just noticed the same issue. This ticket should be reopened.

**Comment by psntr** [comment]
Same issue here

**Comment by notflip** [comment]
@psntr @HumidBrains It's fixed with 3.48.1. @marco-land this can be closed!
